database/sql/postgres/egg-postgres.json

```diff
@@ -22,7 +22,7 @@
     },
     "scripts": {
         "installation": {
-            "script": "#! \/bin\/ash\r\nadduser -D -h \/home\/container container\r\n\r\nchown -R container: \/mnt\/server\/\r\n\r\nsu container -c 'initdb -D \/mnt\/server\/postgres_db\/ -A md5 -U \"$PGUSER\" --pwfile=<(echo \"$PGPASSWORD\")'\r\n\r\nmkdir -p \/mnt\/server\/postgres_db\/run\/\r\n\r\necho -e \"Done\"",
+            "script": "#! \/bin\/ash\r\nadduser -D -h \/home\/container container\r\n\r\nchown -R container: \/mnt\/server\/\r\n\r\nsu container -c 'initdb -D \/mnt\/server\/postgres_db\/ -A md5 -U \"$PGUSER\" --pwfile=<(echo \"$PGPASSWORD\")'\r\n\r\nmkdir -p \/mnt\/server\/postgres_db\/run\/\r\n\r\n## Add default \"allow from all\" auth rule to pg_hba\r\nif ! grep -q \"# Custom rules\" \"\/mnt\/server\/postgres_db\/pg_hba.conf\"; then\r\n    echo -e \"# Custom rules\\nhost    all             all             0.0.0.0\/0               md5\" >> \"\/mnt\/server\/postgres_db\/pg_hba.conf\"\r\nfi\r\n\r\necho -e \"Done\"",
```

Comment:
This should be in the config parser. It runs on each server startup instead of only on installation. Other than that, LGTM.

Reply:
This is a little more difficult than I originally thought.
The only posibility I would have would be to search for "host all all" with the file parser and then replace it with the wanted content (e.g. "host all all 0.0.0.0/0 md5"). 

But even if I put the address into a variable and make it configurable, I would prevent the user from configuring multiple entries to different IP addresses, e.g:
 > host all all 5.5.5/32 md5
 > host all all 4.4.4.4/32 md5

Of course, I don't know if such a customisation would even be widely used ^^
If this restriction is not a problem, I can adjust the Egg accordingly.